solutions/cses-1076.mdx

```diff
@@ -87,3 +82,103 @@ int main() {
 	cout << endl;
 }
 ```
+
+## Solution 2: Fenwick Tree
+
+We can use a [Fenwick tree](/gold/PURS#binary-indexed-tree) to simulate an order statistic tree/indexed set.
```

Comment:
ok this problem is in gold so idt there's a reason to explain how BITs work
maybe just link to the module that explains it and go